recipes/biocode/meta.yaml

```diff
@@ -0,0 +1,141 @@
+{% set name = "biocode" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ab48e0ea92f52c7fe48c6c947d4d5aa43437f16cd42a804546c71ef0e5ba2265
+  patches: biocode.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  
+requirements:
+  host:
```

Comment:
The host section should only contain those requirements needed to run `pip install`, often these are only `pip` and `python`.